udrzba/_pomocne.php

```diff
@@ -70,6 +94,21 @@ function nasad(array $nastaveni) {
   msg('nasazení dokončeno');
 }
 
+function getFilesAlwaysRequiredByAutoloader(): array {
+  if (!file_exists(__DIR__ . '/../vendor/composer/autoload_files.php')) {
+    return [];
+  }
+  $alwaysAutoloadedAbsolute = require __DIR__ . '/../vendor/composer/autoload_files.php';
+
+  return array_map(
+    static function(string $absolutePath) {
+      // create path relative to project root
+      $vendorPosition = strpos($absolutePath, '/vendor/');
+      return substr($absolutePath, $vendorPosition);
+    },
+    $alwaysAutoloadedAbsolute);
```

Comment:
Některé soubory se prostě vždycky incudují při startu autoloaderu, ať už je potřebujeme nebo ne. A když to jsou zrovna soubory z knihoven pro testy, třeba PHPUnit, které na server nekopírujeme, tak kvůli tomu crashne autoloader.

Proto konkrétní soubory, které Composer autoloader bude includoat stůj co stůj, tak musíme na server zkopírovat. K získání jejich seznamu je tahle funkce.